hwilib/devices/bitbox02.py

```diff
@@ -299,13 +267,16 @@ def init(self, expect_initialized: bool = True) -> bitbox02.BitBox02:
     def close(self) -> None:
         self.transport.close()
 
-    def get_master_fingerprint_hex(self) -> str:
+    def get_master_fingerprint(self) -> bytes:
```

Comment:
What is the point of splitting this into two functions?

Reply:
`get_master_fingerprint_hex` is needed to match the api, but it's better to use `get_master_fingerprint` internally.